.github/stale.yml

```diff
@@ -0,0 +1,52 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 14
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels: []
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+
+# Comment to post when removing the stale label.
+# unmarkComment: >
+#   Your comment here.
+
+# Comment to post when closing a stale Issue or Pull Request.
+# closeComment: >
+#   Your comment here.
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 30
+
+# Limit to only `issues` or `pulls`
+# only: issues
```

Comment:
Pulls seem a little more likely to be worked on / tracked in other systems without update activity in Github. I think we want to limit this to `issues`

Reply:
For issues that are tracked in other systems: do they usually have a special label already attached, like the `zube` labels I sometimes see? If so, what are your thoughts on selectively applying this to issues? 
I could be wrong, but I imagine that we have a decent number of issues that aren't tracked elsewhere, and could benefit from being resurrected and/or closed.

Reply:
Is it possible to have this automate the closing of issues, too, but just give them a little more runway? Alternatively, I think that if an issue hasn't seen activity in 90 days, it's fine to close it. Whoever is tracking it externally can re-open it when it becomes important enough to put onto dev's radar, no?

Reply:
We talked about auto closing old issues before and decided no. Not that we can't bring it up again, but there are legit old issues that don't see activity and should still be open.   I think for the sake of this PR we stick to pulls only and can bring up the issues one separately. (I'll forward you all the email thread for historical context)